### ISSUE TITLE
Support for React 17

### ISSUE THREAD

**Issue by ale917k** [samuelmeuli/react-magnifier#19]
Since upgrading to React 17, the package manager flags about a difference in peer dependency version for react:
`react-magnifier@3.0.4" has incorrect peer dependency "react@^16.0.0".`

Can the dependency be updated accordingly? [No new feature changes have been introduced](https://reactjs.org/blog/2020/10/20/react-v17.html), thus no breaking changes can be introduced with the upgrade

**Comment by BearCooder** [comment]
Yes both an uprade to v17 or v18 is not possible because of dependency conflict.
